### PR TITLE
Introduce es6 chai mocha

### PR DESCRIPTION
**Pull request by komalahuwalia**
Before your pull request is reviewed and merged, please ensure that:

* [ ] there are no linting errors
* [ ] your code is in a uniquely-named feature branch and has been rebased on top of the latest master. If you're asked to make more changes, make sure you rebase onto master then too!
* [ ] your pull request is descriptively named and links to an issue number, i.e. `Fixes #123`
* [ ] **Make sure you have run the tests!!**

Thank you!

### CODE REVIEW

File: server/.build/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+*
+*/
```
Comment (MathuraMG) [inline server/.build/.gitignore]: @komalahuwalia  - can we instead include the required files inside the main `.gitignore` ?

File: .vscode/launch.json [hunk 2]
```diff
@@ -5,9 +5,23 @@
             "type": "node",
             "request": "launch",
             "name": "Server Debug",
-            "program": "${workspaceFolder}/server/src/server.js",
+            "program": "${workspaceFolder}/server/.build/server.js",
```
Comment (MathuraMG) [inline .vscode/launch.json]: just curious - does this refer to the file that is built after compilation with babel?

File: server/package.json [hunk 3]
```diff
@@ -2,16 +2,28 @@
...
-    "test": "",
-    "lint": "./node_modules/.bin/eslint ."
+    "lint": "./node_modules/.bin/eslint .",
+    "build": "./node_modules/.bin/babel src --out-dir .build --source-maps",
+    "prestartserver": "./node_modules/.bin/babel src --out-dir .build",
+    "startserver": "node .build/server.js",
+    "prebuild": "rm -rf .build/*",
+    "prestart": "npm run build",
+    "start": "AWS_PROFILE=peblio ./scripts/run_with_credstash.sh nodemon .build/server.js",
+    "test": "./node_modules/mocha/bin/mocha --compilers js:babel-core/register test/**/*.js"
```
Comment (MathuraMG) [inline server/package.json]: Could we add one line in the README about how we can start the test? i.e `npm run test`

File: .vscode/launch.json (hunk 2 above)
Comment (komalahuwalia) [reply .vscode/launch.json]: Yes. This refers to the transpiled file

**Comment by komalahuwalia** [comment]
The Readme about ".build folder + what the new npm tasks are" has been added on another branch https://github.com/peblio/app/pull/562

**Pull request merged**